src/Components/CreateImageWizard/steps/Firewall/components/PortsInput.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { FormGroup } from '@patternfly/react-core';
+
+import { useAppSelector } from '../../../../../store/hooks';
+import {
+  addPort,
+  removePort,
+  selectPorts,
+} from '../../../../../store/wizardSlice';
+import ChippingInput from '../../../ChippingInput';
+import { isPortValid } from '../../../validators';
+
+const PortsInput = () => {
+  const ports = useAppSelector(selectPorts);
+
+  return (
+    <FormGroup label="Ports">
+      {' '}
```

Comment:
What's this for?

Reply:
I probably left a trailing space after `FormGroup` and linter created this beautiful thing when fixing the issue. Will remove it :sweat_smile: